packages/components/Badge/src/Badge.types.ts

```diff
@@ -17,6 +19,12 @@ export type BadgeColor = BadgeNamedColor | ColorValue;
 export type BadgeIconPosition = 'before' | 'after';
 
 export interface BadgeConfigurableProps {
+  /**
+   * A Badge can have its content and borders styled for greater emphasis or to be subtle.
+   * It can be filled, outline, ghost, inverted
+   * @defaultvalue filled
+   */
+  appearance?: BadgeAppearance;
```

Comment:
Why is appearance a token?

Reply:
As it was used in the styling, I needed it for condition. But now I don't need it, because I moved it to lookup function